mm2src/trading_api/src/one_inch_api/classic_swap_types.rs

```diff
@@ -218,9 +219,18 @@ pub struct TokenInfo {
     pub eip2612: bool,
     #[serde(rename = "isFoT", default)]
     pub is_fot: bool,
-    #[serde(rename = "logoURI", with = "serde_one_inch_link")]
-    pub logo_uri: String,
+    #[serde(
+        rename = "logoURI",
+        default,
+        deserialize_with = "serde_one_inch_link::deserialize_opt_string"
+    )] // Note: needed to use 'default' with 'deserialize_with' to allow optional 'logoURI'
+    pub logo_uri: Option<String>,
```

Comment:
doesn't seem to be the case (the need for `default`) with other fields down there that also use `deserialize_with = "serde_one_inch_link::deserialize_opt_string"`...